tests/test_jointcal_cfht.py

```diff
@@ -172,6 +172,16 @@ def test_jointcalTask_2_visits_constrainedPhotometry_no_rank_update(self):
 
         self._testJointcalTask(2, None, None, pa1, metrics=metrics)
 
+    def test_jointcalTask_2_visits_constrainedPhotometry_lineSearch(self):
+        """Activating the line search should only slightly change the chi2 in this case."""
```

Comment:
Hmm, can this expected change be formalized? Checking exact values doesn't seem particularly robust.

Reply:
I don't know how I would go about formalizing the difference here: the values in these tests are almost all empirical (see the note in the readme). I did add a hopefully clarifying comment.